sample.env

```diff
@@ -1,3 +1,4 @@
 NODE_ENV=development
 PORT=8080
 SESSION_SECRET=xxxx-xxxx-xxxx-xxxx-xxxx
+CLAIM_BASE_URL=http://localhost:8090
```

Comment:
Can we rename this to CLAIMANT_SERVICE_URL for clarity, took me a bit to figure out what this was really for.